zbx_nginx_stats.py

```diff
@@ -1,31 +1,50 @@
-#!/usr/bin/python
+#!/usr/bin/env python
 
-import urllib2, base64, re, struct, time, socket, sys, datetime, os.path
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import sys
+is_python_3 = sys.version_info > (3, 0)
+
+import argparse, base64, re, struct, time, socket, datetime, os.path
+
+if is_python_3:
```

Comment:
It's better use `try...except` to support both versions of python.

Something like that:
```python
# For Python 2 and 3 compatibility
try:
    import urllib2
except ImportError:
    # Since Python 3, urllib2.Request and urlopen were moved to
    # the urllib.request.
    import urllib.request as urllib2
```